lib/tweet-webhook.js

```diff
@@ -0,0 +1,41 @@
+"use strict";
+const privateConfig = require("../private-config.json");
+const { standardFromRepo } = require("./helpers/workstreams.js");
+const composeTweet = require("./compose-tweet.js");
+const Twitter = require("twitter-lite");
+
+module.exports = async body => {
+  if (body.ref !== "refs/heads/main") {
+    return;
+  }
+
+  const standard = standardFromRepo(body.repository.name);
+  if (!standard) {
+    return;
+  }
+
+  const { twitter } = standard;
+  const keys = privateConfig.twitterAccounts[twitter];
+  if (!keys) {
+    return;
+  }
+  const client = new Twitter({
+    consumer_key: privateConfig.twitterBot.key,
+    consumer_secret: privateConfig.twitterBot.secret,
+    access_token_key: keys.key,
+    access_token_secret: keys.secret
+  });
+
+  for (const commit in body.commits) {
+    // The composeTweet function either returns the tweet status or null if the commit does not
+    // need a tweet. Given that the tweet status is composed of maximum 70 characters, plus a thank
+    // you of around 30 characters, depending on the name of the author, and the commit URL which
+    // will be shortened, it should not exceed 280 characters.
```

Comment:
This comment feels out of place here; probably it belongs in composeTweet?

Reply:
The reason I put it here is because this file invokes the Twitter API which has the limitations. So it seemed better to document here why we would not expect it to return an error.